quacc/recipes/qchem/core.py

```diff
@@ -0,0 +1,103 @@
+"""
+Core recipes for the Q-Chem
+"""
+from __future__ import annotations
+
+from copy import deepcopy
+from typing import Literal
+
+from ase.atoms import Atoms
+from monty.dev import requires
+
+from quacc.schemas.ase import (
+    summarize_opt_run,
+    summarize_run,
+    summarize_thermo_run,
+    summarize_vib_run,
+)
+from quacc.util.calc import run_ase_opt, run_ase_vib, run_calc
+from quacc.util.thermo import ideal_gas
+from quacc.calculators.qchem import QChem
+
+def static_job(
+    atoms: Atoms,
+    qchem_kwargs: dict,
+) -> dict:
+    """
+    Carry out a single-point calculation.
+
+    Parameters
+    ----------
+    atoms
+        Atoms object
+    qchem_kwargs
+        Dictionary of kwargs for the Q-Chem calculator.
+
+    Returns
+    -------
+    dict
+        Dictionary of results from quacc.schemas.ase.summarize_run
+    """
+    input_atoms = deepcopy(atoms)
+
+    calc = QChem(atoms, **qchem_kwargs)
+    atoms.calc = calc
+    atoms = run_calc(atoms)
+    return summarize_run(
+        atoms,
+        input_atoms=input_atoms,
+        additional_fields={"name": "Q-Chem Static"},
+    )
+
+
+def opt_job(
```

Comment:
For better or for worse, the convention I've gone with is `relax_job` for structure/geometry relaxations, so let's stick with it for consistency's sake.